apps/www/lib/authors.json

```diff
@@ -421,5 +421,13 @@
     "position": "Engineering",
     "author_url": "https://github.com/imor",
     "author_image_url": "https://github.com/imor.png"
+  },
+  {
+    "author_id": "timabdulla",
+    "author": "Tim Abdulla",
+    "username": "timabdulla",
+    "position": "Guest Author",
+    "author_url": "https://github.com/timabdulla",
+    "author_image_url": "https://github.com/timabdulla.png"
```

Comment:
json parsing struggling here for some reason https://vercel.com/supabase/zone-www-dot-com/GUk92NyPsmEsheqspFLJVbbdHJsL

Reply:
![CleanShot 2024-03-01 at 00 05 30@2x](https://github.com/supabase/supabase/assets/37541088/0b5b04ba-4e78-41b9-aca6-c0883b35aede)
looks like vercel has a trailing comma in the source but I checked out the branch locally and I don't see it so I cant fix it :/